### PR TITLE
Feature/big chat icon new

### PR DESCRIPTION
**Pull request by MathiasOhrstrom**

### CODE REVIEW

**Comment by hedvigoscar** [comment]
Pull Request build: https://app.bitrise.io/artifact/48941144/p/515d0d46961d455cdc3792fabc1f3884

**Review by hedvigoscar — APPROVED** [review]

File: app/src/main/res/drawable/divider.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape android:shape="rectangle" xmlns:android="http://schemas.android.com/apk/res/android">
     <size android:height="1dp"/>
-    <solid android:color="@color/separator_on_surface"/>
```
Comment (hedvigoscar) [inline app/src/main/res/drawable/divider.xml:4]: Where is this one used? Do we risk an undesirable regression with this change?

**Review by hedvigoscar — COMMENTED** [review]
One slight concern. Auto-approved once you've investigated it

File: app/src/main/res/drawable/divider.xml (hunk 1 above)
Comment (MathiasOhrstrom) [reply app/src/main/res/drawable/divider.xml:4]: The layouts effected looks as expected :)

**Pull request merged**